x-pack/plugins/aiops/public/components/log_categorization/show_flyout.tsx

```diff
@@ -45,15 +44,10 @@ export async function showCategorizeFlyout(
       };
 
       const appDependencies: AiopsAppDependencies = {
-        analytics,
-        notifications,
-        uiSettings,
-        http,
-        theme,
-        application,
-        i18n,
+        ...coreStart,
         ...plugins,
       };
+      const startServices = pick(coreStart, 'analytics', 'i18n', 'theme');
```

Comment:
The changes in this PR only use `lodash.pick` in files where it's already imported. Otherwise, it considers that it doesn't matter to the SharedUX module if more CoreStart fields are provided than truly necessary.